install

```diff
@@ -162,8 +162,8 @@ group_chmods = %w( bin etc Frameworks include lib sbin share var
                    map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
 # zsh refuses to read from these directories if group writable
 zsh_dirs = %w( share/zsh share/zsh/site-functions )
-user_chmods = zsh_dirs.
-                  map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
+                  .map { |d| File.join(HOMEBREW_PREFIX, d) }
```

Comment:
Ok, in that case you want to`&&` to the `chmod?` logic rather than removing it to check for additional group permissions.